utils/cli/src/table.rs

```diff
@@ -4,8 +4,8 @@ use std::collections::HashMap;
 
 pub fn print_table(
     columns: &[String],
-    values: &Vec<serde_json::Value>,
-    summary: &Vec<serde_json::Value>,
+    values: &[serde_json::Value],
+    summary: &[serde_json::Value],
```

Comment:
Initially I wanted to update Rust to `1.76` but Clippy produced too many issues.
I missed reverting this one, but it does no harm.